rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -193,10 +198,123 @@ public void registerBtcCoinbaseTransaction_after_RSKIP143_activation_null_data()
+        Transaction rskTxMock = mock(Transaction.class);
+        doReturn(new RskAddress(key.getAddress())).when(rskTxMock).getSender();
+
+        Bridge bridge = getBridgeInstance(rskTxMock, bridgeSupportMock, activations);
+
+        try {
+            executor.execute(bridge, null);
+            fail("VMException should be thrown!");
+        } catch (VMException vme) {
+            Assert.assertEquals("Sender is not part of the active or retiring federations, so he is not enabled to call the function 'null'", vme.getMessage());
+        }
+    }
+
+    @Test
+    public void activeAndRetiringFederationOnly_activeFederationIsFromFederateMember_OK() throws Exception {
+        BridgeMethods.BridgeMethodExecutor decorate = mock(BridgeMethods.BridgeMethodExecutor.class);
+        BridgeMethods.BridgeMethodExecutor executor = Bridge.activeAndRetiringFederationOnly(decorate, null);
+
+        ActivationConfig activations = spy(ActivationConfigsForTest.genesis());
+        doReturn(true).when(activations).isActive(eq(RSKIP143), anyLong());
+
+        Federation activeFederation = new Federation(FederationTestUtils.getFederationMembersFromPks(100, 200, 300, 400, 500, 600), ZonedDateTime.parse("2017-06-10T02:30:01Z").toInstant(), 0L, NetworkParameters.fromID(NetworkParameters.ID_REGTEST));
+
+        BridgeSupport bridgeSupportMock = mock(BridgeSupport.class);
+        doReturn(activeFederation).when(bridgeSupportMock).getActiveFederation();
+        doReturn(null).when(bridgeSupportMock).getRetiringFederation();
+
+        // RSK Address is constructed by FederationTestUtils as ECKey.fromPrivate(BigInteger.valueOf(PK + 1))
+        ECKey key = ECKey.fromPrivate(BigInteger.valueOf(101));
+        Transaction rskTxMock = mock(Transaction.class);
+        doReturn(new RskAddress(key.getAddress())).when(rskTxMock).getSender();
+
+        Bridge bridge = getBridgeInstance(rskTxMock, bridgeSupportMock, activations);
+
+        executor.execute(bridge, null);
+
+        verify(bridgeSupportMock, times(1)).getActiveFederation();
+        verify(decorate, times(1)).execute(any(), any());
+    }
+
+    @Test
+    public void activeAndRetiringFederationOnly_activeFederationIsNotFromFederateMember_retiringFederationIsNotNull_retiringFederationIsFromFederateMember_OK() throws Exception {
+        BridgeMethods.BridgeMethodExecutor decorate = mock(BridgeMethods.BridgeMethodExecutor.class);
+        BridgeMethods.BridgeMethodExecutor executor = Bridge.activeAndRetiringFederationOnly(decorate, null);
+
+        ActivationConfig activations = spy(ActivationConfigsForTest.genesis());
+        doReturn(true).when(activations).isActive(eq(RSKIP143), anyLong());
+
+        Federation activeFederation = new Federation(FederationTestUtils.getFederationMembersFromPks(100, 200, 300, 400, 500, 600), ZonedDateTime.parse("2017-06-10T02:30:01Z").toInstant(), 0L, NetworkParameters.fromID(NetworkParameters.ID_REGTEST));
```

Comment:
Please indent your code. We configured intellij to hard wrap at line 100 and we intend to wrap it by level (if you need to wrap one parameter, you wrap one by one). This is a example of what I mean:
```suggestion
        Federation activeFederation = new Federation(
            FederationTestUtils.getFederationMembersFromPks(100, 200, 300, 400, 500, 600), 
            ZonedDateTime.parse("2017-06-10T02:30:01Z").toInstant(), 
            0L, 
            NetworkParameters.fromID(NetworkParameters.ID_REGTEST)
        );
```

Reply:
Done!